app/models/history.rb

```diff
@@ -39,6 +39,13 @@ class History < ApplicationRecord
 
   belongs_to :patient
 
+  after_save :touch_patient
+  before_destroy :touch_patient
+
+  def touch_patient
+    patient.touch unless history_type == HISTORY_TYPES[:report_reminder]
```

Comment:
This is difficult but we seem to use various History types for this kind of thing which is a consistency issue. In both the `PatientMailer` and the `ConsumeAssessmentJobs, we update a patient record with history items of type `Contact Attempt` if there's an issue as well which we wouldn't want to be included as a touch based on our criteria. It doesn't look like we use `Contact Attempt` history types for anything the user does so many this just needs to be updated to check both `report_reminder` and `contact_attempt` history types.

Reply:
I think we have to account for manual contact attempts:

![Screen Shot 2021-02-10 at 1 59 29 PM](https://user-images.githubusercontent.com/9956561/107557943-36066a80-6ba8-11eb-9cbb-3da97a29774a.png)

so I just checked for `created_by` to be `Sara Alert System`: https://github.com/SaraAlert/SaraAlert/pull/683/commits/7bc9ddfc5b56b8c866b3e3201e438225a24315b6

Reply:
Good point. Would you mind adding a comment around that line that explains the logic?

Reply:
sure! https://github.com/SaraAlert/SaraAlert/pull/683/commits/f08bac4ad3811e049e75476f487dacacf231f40f

Reply:
Isn't this still missing the export case?

Reply:
@Bialogs are you talking about the `:monitoree_data_download` history_type?

Reply:
Right good catch @Bialogs - I think we need to exclude that history type as well @timwongj

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/683/commits/1987337042767c09797d9cb63d699f585d003f30

Reply:
Thanks, we should also have tests for these cases specifically in the related test file

Reply:
https://github.com/SaraAlert/SaraAlert/pull/683/commits/1abe7f833a84aaf78e5f96a5cd1aeca53fcb80c3